RESOURCES.md

```diff
@@ -1,4 +1,6 @@
-# Resources for the Puppet Module for HPE OneView
+# Summary
+
+## HPE OneView resources
```

Comment:
Aesthetically, isn't good to have a link in a heading.
Specially in this case, it'll not be linked to any specific content.